Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int x = 2;
+            while (x > 0)
+            {
+                //Thank you for reviewing my code. You guys are amazing.
+
+                Console.Write("Hello. What is your name? ");
+                string name = Console.ReadLine();
+
+                Console.Write("What is your last name? ");
+                string surname = Console.ReadLine();
+
+                Console.Write("How old are you? ");
+                string age = Console.ReadLine();
+
+                Console.Write("How much do you weigh? ");
+                string weight = Console.ReadLine();
+
+                Console.Write("How tall are you? ");
+                string height = Console.ReadLine();
+
+                double bmi = Convert.ToInt32(weight) / Convert.ToInt32(height) ^ 2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'bmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4958692)